supertokens_python/recipe/session/session_class.py

```diff
@@ -30,7 +30,7 @@ async def revoke_session(self, user_context: Union[Any, None] = None) -> None:
         await self.recipe_implementation.revoke_session(
             self.session_handle, user_context
         )
-        self.remove_cookies = True
+        self.remove_tokens = True
```

Comment:
you don't need this either cause you can just add to the `methods_to_call` list.